ports/stm32/main.c

```diff
@@ -528,6 +529,10 @@ void stm32_main(uint32_t reset_mode) {
     machine_i2s_init0();
     #endif
 
+    #if MICROPY_HW_ENABLE_SUBGHZ_RADIO
+    machine_subghz_radio_init();
```

Comment:
This would be called `stm_subghz_radio_init()`.

Reply:
I went with `subghz_init()`, in keeping with the naming used for the `rfcore` subsystem and the suggested prefix for the Python functions.